week5/src/InputMismatchException/inputmismatch.java

```diff
@@ -0,0 +1,37 @@
+package InputMismatchException;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+
+public class inputmismatch{
+
+    public static void main(String[] args) {
+        int num1 = 0, num2=0;
+        Scanner  scan =new Scanner( System .in);
+
+        while (true)
```

Comment:
why use while? you only need it two times one if the first time int try block and the second in catch block if there is an exception